_includes/components/gc-minister/gc-minister.html

```diff
@@ -0,0 +1,22 @@
+<section class="gc-minister">
+	<h2>{% if page.language == "en" %}Minister(s){% else %}Ministre(s){% endif %}</h2>
+	<div class="row wb-eqht-grd">
+		<div class="col-md-6">
+			<div class="row">
+				<div class="pull-right col-md-7 col-xs-12">
+					<h3><a href="#">{% if page.language == "en" %}The Honourable [Minister name]{% else %}L'honorable [prénom et nom de famille]{% endif %}</a></h3>
+				</div>
+				<div class="col-md-5 col-xs-12">
+					<img src="./img/minister-200x250.jpg" alt="{% if page.language == 'en' %}The Honourable [Minister name]{% else %}L'honorable [prénom et nom de famille]{% endif %}">
+				</div>
+				<div class="col-md-7 col-xs-12">
+					<p>{% if page.language == "en" %}[Official title]{% else %}[Titre officiel du ministre]{% endif %}</p>
```

Comment:
Do they want a different column configuration for this one? May be it can take the full with but we do apply the content width limited rule?

Reply:
Please view this comment: https://github.com/wet-boew/GCWeb/pull/2225#issuecomment-1864835226

Reply:
As discussed verbally with @duboisp, add instruction on the sizing in notes. Also, add "mediaConsiderations" property in changeset.